fairscale/experimental/wgit/pygit.py

```diff
@@ -62,14 +84,23 @@ def _init_wgit_git(self, gitignore: List) -> None:
                 file.write(f"{item}\n")
 
     def add(self) -> None:
-        """git add all the untracked files not in gitignore, to the .wgit/.git repo."""
+        """
+        git add all the untracked files not in gitignore, to the .wgit/.git repo.
+        """
         # If .wgit is git repo, add all the files in .wgit not being ignored to git
+        # TODO: Add functionalities for add specific files and add all files.
         if self._exists:
```

Comment:
in which case will this if condition fail? if it never fails, turn this into an assert? if it does fail, we don't need to do anything in the "else" case? Add a comment.